src/mlpack/core/cv/simple_cv.hpp

```diff
@@ -0,0 +1,197 @@
+/**
+ * @file simple_cv.hpp
+ * @author Kirill Mishchenko
+ *
+ * A simple cross-validation strategy.
+ *
+ * mlpack is free software; you may redistribute it and/or modify it under the
+ * terms of the 3-clause BSD license.  You should have received a copy of the
+ * 3-clause BSD license along with mlpack.  If not, see
+ * http://www.opensource.org/licenses/BSD-3-Clause for more information.
+ */
+#ifndef MLPACK_CORE_CV_SIMPLE_CV_HPP
+#define MLPACK_CORE_CV_SIMPLE_CV_HPP
+
+#include <mlpack/core/cv/meta_info_extractor.hpp>
+#include <mlpack/core/cv/cv_base.hpp>
+
+namespace mlpack {
+namespace cv {
+
+/**
+ * SimpleCV splits data into two sets - training and validation sets - and then
+ * runs training on the training set and evaluates performance on the validation
+ * set.
+ *
+ * To construct a SimpleCV object you need to pass the validationSize parameter
```

Comment:
Looks great to me.  Can you wrap the code in `@code` and `@endcode` for doxygen?  It might be good to randomly generate the dataset instead of using `...` for clarity, if you like:

```
// 100-point 5-dimensional random dataset.
arma::mat data = arma::randu<arma::mat>(5, 100);
// Random shuffled labels in [0, 5).
arma::Row<size_t> labels = arma::shuffle(arma::linspace<arma::Row<size_t>>(0, 4, 100));
size_t numClasses = 5;
```